release-20.2: ui: fix aggregation on txn details

vladlos opened the pull request:
in old version admin-ui-components we have known issue on txn
datails page with duplications of stmns displayed on multinode
clusters. as admin-ui-components is deprecated switched this dependency on relative version of cluster-ui.

as 20.2 console db pointed to pretty old commit in yarn-vendored
master-20.2 was created and contains corresponding changes for
current pr.

Release note (ui): fix duplicates of statements on transactions details page for multi-node clusters

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/62797)
<!-- Reviewable:end -->

nathanstilwell reviewed (approved):
> Reviewed 15 of 15 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/62797)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @elkmaster)



<!-- Sent from Reviewable.io -->

nathanstilwell commented:
The changes look good and testing locally looks solid. @elkmaster I'm going to look into this test failure in the morning.

vladlos committed:
ui: fix aggregation on txn details

in old version admin-ui-components we have known issue on txn
datails page with duplications of stmns displayed on multinode
clusters. as admin-ui-components is deprecated switched this
dependency on relative version of cluster-ui.

as 20.2 console db pointed to pretty old commit in yarn-vendored
master-20.2 was created and contains corresponding changes for
current pr.

Release note (ui): fix duplicates of statements on transactions
details page for multi-node clusters
